frontend/packages/core/src/Input/toggle-button-group.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  ToggleButton as MuiToggleButton,
+  ToggleButtonGroup as MuiToggleButtonGroup,
+} from "@material-ui/lab";
+
+export interface ToggleButtonGroupProps {
+  exclusive?: boolean;
+  currentValue: string;
+  onChange: (event: React.ChangeEvent<{}>, value: string) => void;
+  toggleButtonValues: string[];
+}
+
+const StyledMuiToggleButtonGroup = styled(MuiToggleButtonGroup)({
+  ".MuiToggleButton-root": {
+    "&.Mui-selected": {
+      color: "#3548D4",
+      backgroundColor: "#3548D41F",
+    },
+  },
+  padding: "6px 16px 16px 6px",
+});
+
+// TODO(smonero): add some tests
+const ToggleButtonGroup = ({
+  exclusive = true,
```

Comment:
what does exclusive represent?

Reply:
the exclusive represents whether or not there can be only one option selected at a time. If exclusive is false, you can select multiple options simultaneously (ex. think about editing text, where you can select both bold and italic at the same time). If exclusive is true, user is forced to only select one

Reply:
I think it might be a little more intuitive to use something like `multiple?: boolean` which should default to false. That way it works as expected for the base case and if you need multi-select you could optionally set it.